deploy/olm-catalog/ibm-block-csi-operator-community/1.5.0/bundle-1.5.0.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator-community
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.6,v4.7"
```

Comment:
no need `v4.6-v4.7`?

Reply:
They approved it. I don't want to change it now.

Reply:
ok, I hope we wont have any trouble in the future